python/ql/lib/semmle/python/security/dataflow/LogInjectionCustomizations.qll

```diff
@@ -0,0 +1,65 @@
+/**
+ * Provides default sources, sinks and sanitizers for detecting
+ * "command injection"
```

Comment:
This file says `"command injection"` all over the place. We need to replace that with `"log injection"` :+1:

Reply:
:doh: